### DISCUSSION TITLE
Client connection

### DISCUSSION THREAD

**Discussion by MortenTobiasNielsen** [uNetworking/uWebSockets#1224] [Q&A] [answered]
Hi, 

It seems like uWebSockets are the very best and I therefore really want to use it. I am very new to websockets and find it hard to wrap my head around. Currently I am not in need of a websocket server but instead want to connect to a websocket. 

I want to connect to wss://api2.example.com how would I do this with uWebSockets?

**Comment by ghost** [comment, marked as answer]
Hello, there are a lot better resources to start out with than this repo. Mozilla has some great documentation of WebSockets

**Reply by MortenTobiasNielsen** [reply]
Okay, I will have a look at their documentation. 

Still out of curiocity - How would a client connect to uWebSocket server?

**Reply by ghost** [reply]
I'm not a teacher. You have excellent material to read by people who can explain it 10x better than I can.

**Reply by dmpeletronic** [reply]
Hello @alexhultman and @MortenTobiasNielsen..
I'm on the same situation as @MortenTobiasNielsen.. I'm trying to find a working example of uWebSocket as a client connecting to some uri as mentioned.

Where do I find a simple example of uWebSocket as a client in cpp?

best regards

**Reply by ghost** [reply]
It doesn't have client support. Only server.

**Comment by HamiltonWang** [comment]
no client support??????

**Reply by uNetworkingAB** [reply]
No client support as of now

**Reply by jeasonche09** [reply]
@uNetworkingAB we neeeeeeeeeeeed client, pls.
